lib/Conversion/TorchToLinalg/DataMovement.cpp

```diff
@@ -1179,12 +1179,26 @@ class ConvertAtenBroadcastToOp : public OpConversionPattern<AtenBroadcastToOp> {
       return rewriter.notifyMatchFailure(
           op, "unimplemented: the size list is not from list construct");
     }
+    SmallVector<bool> shiftToBroadcastShape;
```

Comment:
I'm not sure I understand what this represents. Can you explain to me the reasoning?

Reply:
Hi @ramiro050 ,

1. I've now renamed this to `useBroadcastToShape`.
2. `broadcastToShape` is already a construct which is/was previously being used in the lowering.
3. Now with `useBroadcastToShape` I'm trying to use `broadcastToShape` only for those dimensions which are dynamic and whose final shape needs to be inferred based on the "shape being broadcastedTo".

I've added a comment too in the file to address this.

Let me know if that sounds good to you.